app/javascript/src/components/Dashboard/Notes/Pane/utils.js

```diff
@@ -0,0 +1,16 @@
+import { buildSelectOption } from "utils";
+
+export const parseNoteValues = values => {
+  const assignedContact = values.assignedContact.label;
+  const tags = values.tags.map(tag => tag.label);
+
+  return { ...values, tags, assignedContact };
+};
+
+export const buildNoteInitialValues = note => ({
+  ...note,
+  assignedContact: note.assignedContact
+    ? buildSelectOption(note.assignedContact)
+    : null,
```

Comment:
Go through https://academy.bigbinary.com/learn-react/react-best-practices/best-practices-associated-with-conditional-statements very thoroughly.

Reply:
Sorry, force of habbit. I was aware of short-circuiting, I will modify the code accordingly. Thanks for pointing it out.